website/components/ck-hero/index.jsx

```diff
@@ -0,0 +1,44 @@
+import Button from '@hashicorp/react-button'
+import s from './style.module.css'
+
+export default function CKHero({ title, subtitle, ctas, media }) {
+  return (
+    <div
+      className={s.ckHero}
+      style={{
+        backgroundImage: `url(${require('./img/background-design.svg')})`,
+      }}
+    >
+      <div className="g-grid-container">
+        <div className={s.headline}>
+          <h1 className={s.title}>{title}</h1>
+          <p className={s.subtitle}>{subtitle}</p>
+          {ctas.map(({ text, url }, idx) => (
+            <Button
+              key={text}
+              theme={{
+                brand: idx === 0 ? 'consul' : 'neutral',
+                variant: 'primary',
+              }}
+              linkType={idx === 0 ? null : 'inbound'}
+              url={url}
+              title={text}
+              className={idx === 0 ? null : s.inboundButton}
```

Comment:
You can actually get the presentation variation you are looking for that you're solving via css with the `inboundButton` with this set of props.

```jsx
<Button
  title='Button Text'
  linkType="inbound"
  theme={{
   "variant": "tertiary-neutral",
   "background": "dark",
  }}
/>
```